data/campaigns/Secrets_of_the_Ancients/scenarios/07_Meeting_of_the_Minds.cfg

```diff
@@ -234,7 +234,7 @@ The tunnel at the back of the cave narrowed and ran on for quite some time. Patc
         [/message]
         [message]
             speaker=Ras-Tabahn
-            message= _ "Those goblins were too weak, and it would take their spirits a long time to become strong enough to be useful. You are going to raise Elvish warriors instead. A few miles into the Grey Woods, there lies an ancient battleground from a civil war hundreds of years ago. You will follow an old elf road there and raise an army of ghosts while I visit Carcyn, a nearby town. We will meet at the end of the elf road by the Great River, and I will explain where we should go next."
+            message= _ "Those goblins were too weak, and it would take their spirits a long time to become strong enough to be useful. You are going to raise elvish warriors instead. A few miles into the Grey Woods, there lies an ancient battleground from a civil war hundreds of years ago. You will follow an old elf road there and raise an army of ghosts while I visit Carcyn, a nearby town. We will meet at the end of the elf road by the Great River, and I will explain where we should go next."
```

Comment:
I just realized this line has "miles" in it which is probably not a great way to describe distance in the given time frame. Out of the scope of this PR though.

Reply:
Yes, out of scope - but out of curiosity what is the distance convention? Measurement by march or horse riding time? I didn't notice it was out of place till you mentioned it.

If you're going to change that, a quick search shows 18 instances of 'miles' in campaign prose.

Reply:
Miles are an ancient unit, with many possible values.  The roman mile was simply a thousand paces (mille passus), though a roman 'pace' consisted of two steps.

It only sounds out of place because americans (and the brits) and international shipping still use it.

Reply:
For what it's worth, I'm fine with miles. It could be a specific distance as used in a few countries today, or as an arbitrary term such as 'marching for miles and miles'. On the other hand, the kilometre would sound very much out of place in Wesnoth.

Feel free to continue the conversation but I'm marking this as resolved as it's separate to this PR.

Reply:
Oki doki, leave it then.